src/lib.rs

```diff
@@ -383,15 +383,15 @@ pub enum OperandPos {
 /// that the conflict (overlap) is properly accounted for. See
 /// comments on the constructors below for more.
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+//#[repr(packed)]
```

Comment:
```suggestion
```

Reply:
Oops that's from an old test I was doing.